.circleci/config.yml

```diff
@@ -187,8 +187,8 @@ jobs:
           SRCDIR: /src/github.com/weaveworks/wksctl
           KUBECTL_URL: https://dl.k8s.io/v1.10.5/kubernetes-client-linux-amd64.tar.gz
           KUBECTL_CHECKSUM: da9d557989a0b9671a610f21642052febb8f70c3cf144c98a8a4f7ecab6bafe2
-          CREATE_IMAGE: 0
-          USE_IMAGE: 0
+          CREATE_IMAGE: 1
```

Comment:
What was this supposed to do?  You took out the call to the place it gets used.

Reply:
we thought we used the images created but they never got used by terraform